### PR TITLE
maintainers: remove expipiplus1

### PR DESCRIPTION
**Pull request by expipiplus1**
Another year, another exhausting undermoderated thread full of people arguing against being an inclusive community, tone policing and denying that discriminatory behaviour even exists.

This clearly isn't getting any better and I'm embarrassed that I stayed around for this long. I can't continue to lend my name to such a project.

### CODE REVIEW

**Review by tomodachi94 — APPROVED** [review]
Diff lgtm, you will be missed ❤️

Can you resolve the merge conflict that is occuring? It's at `pkgs/applications/graphics/xrgears/default.nix`, which moved to `pkgs/by-name/xr/xrgears/package.nix`.

**Comment by tomodachi94** [comment]
Thank you.

I hope the community improves to the point where you'll want to come back (for both of our sakes). Thank you for maintaining these packages ❤️ 

Packages orphaned by this, for the benefit of prospective maintainers:
* `signald`
* `fped`
* `libgpiod`
* `libgpiod_1`
* `epcdap`
* `pangolin`
* `swiftshader`
* `mx-puppet-discord`
* `vulkan-extension-layer`
* `vulkan-tools-lunarg`
* `inav-blackbox-tools`
* `opencorsairlink`
* Some `haskellPackages` packages:
    - `VulkanMemoryAllocator`
    - `autoapply`
    - `exact-real`
    - `language-c`
    - `orbits`
    - `vector-sized`
    - `vulkan`
    - `vulkan-utils`

File: maintainers/team-list.nix [hunk 1]
```diff
@@ -449,7 +449,6 @@ with lib.maintainers;
   haskell = {
     members = [
       cdepillabout
-      expipiplus1
       maralorn
       ncfavier
       sternenseemann
```
Comment (wolfgangwalther) [inline maintainers/team-list.nix:454]: It seems like this change has not been reflected in the Nixos/haskell team memberships: https://github.com/orgs/NixOS/teams/haskell

(same for @peti who is a member of the GitHub team, but not mentioned in the `team-list.nix` file)

@maralorn @sternenseemann is that on purpose? I think we should sync the two lists, in either direction.

Comment (maralorn) [reply maintainers/team-list.nix:454]: Well, for peti, this was basically on purpose. I’d say he is kind of an honorary member of the team so we didn’t want to take away his rights to all the repos which he initiated.

I am actually not completely sure what the list here does. I think removing expipiplus1 on both sides seems sensible.

**Pull request merged**